src/sidebar/components/Thread.js

```diff
@@ -108,11 +109,22 @@ function Thread({ thread, threadsService }) {
         {annotationContent}
 
         {showHiddenToggle && (
-          <div className="Thread__hidden-toggle-button-container">
-            <LabeledButton onClick={() => threadsService.forceVisible(thread)}>
-              Show {countHidden(thread)} more in conversation
-            </LabeledButton>
-          </div>
+          <>
+            {!thread.parent && (
+              <AnnotationHeaderWrapper
```

Comment:
Might I suggest choosing a name that indicates a little more clearly what this component is responsible for? Perhaps `HiddenAnnotationHeader` or `ThreadHeader`? The latter (`ThreadHeader`) might be helpful to highlight that the metadata rendered within pertains to the entire thread...

Reply:
I agree, thanks.